src/neuroglancer/mesh/backend.ts

```diff
@@ -243,8 +244,8 @@ export abstract class MeshSource extends ChunkSource {
     fragmentSource.meshSource = this;
   }
 
-  getChunk(objectId: Uint64) {
-    const key = getObjectKey(objectId);
+  getChunk(objectId: Uint64, clipBounds?: Bounds) {
```

Comment:
Storing the clipBounds in the ManifestChunk will allow the download function to have access to them.